index.js

```diff
@@ -1,42 +1,40 @@
 function stringIncludes(haystack, needle) {
   for (let i = 0; i < haystack.length; i++) {
-    if (haystack.slice(i, needle.length) === needle) {
-      return true
+    if (haystack.slice(i, i + needle.length) === needle) {
+      return true;
     }
   }
 
-  return false
+  return false;
 }
 
 function countLetter(haystack, needle) {
-  let count = 0
-
-  while (haystack.length) {
-    if (haystack.shift() === needle) {
-      count++
+  let count = 0;
+  for (let i = 0; i < haystack.length; i++) {
+    if (haystack.slice(i, i + needle.length) === needle) {
+      count++;
     }
   }
-
-  return count
+  return count;
 }
 
 function camelCase(sentence) {
-  const arr = sentence.split(' ')
+  const arr = sentence.split(" ");
 
-  for (let i = 0; i < arr.length; i++) {
-    arr[i] = arr[i].toLowerCase()
+  for (let i = 1; i < arr.length; i++) {
+    arr[0] = arr[0].toLowerCase();
+    arr[i] = arr[i].toLowerCase();
+    const letters = arr[i].split("");
+    letters[0] = letters[0].toUpperCase();
 
-    const letters = arr[i].split('')
-    letters[0] = letters[0].toUpperCase()
-
-    arr[i] = letters.join('')
+    arr[i] = letters.join("");
```

Comment:
Semicolons and double quotes? Make sure your linter is working, these should have been flagged.